backend/src/org/jetbrains/demo/thinkter/Application.kt

```diff
@@ -50,3 +51,15 @@ fun Application.main() {
     }
 }
 
+class GsonConverter(private val gson: Gson = Gson()) : ContentConverter {
```

Comment:
This is already done, isn't it ? It should be in `ktor-gson`

Reply:
You're right, fixed this in d4b3be9c5b1b620bdecd9e3f59abb839cd4ed4be